pkgs/top-level/beam-packages.nix

```diff
@@ -41,6 +41,15 @@ in
 
     # Standard Erlang versions, using the generic builder.
 
+    erlang_27-rc2 = self.beamLib.callErlang ../development/interpreters/erlang/27-rc2.nix {
+      wxGTK = wxGTK32;
+      parallelBuild = true;
+      autoconf = buildPackages.autoconf269;
```

Comment:
I saw notes about this when `erlang_26` was added in #232278 but unsure if we need to change this or want to